### DISCUSSION TITLE
Does it make sense to make the App ID a config on the `local.properties`?

### DISCUSSION THREAD

**Discussion by dubdabasoduba** [opensrp/fhircore#1611] [Q&A]
- We currently support specific clients so by the time we are sharing the APK we already know the App ID it would use. If we had the APP ID on the `local.properties` then we could auto-populate it and just move to the login page. 
- if the App ID is not provided then we show the APP ID config screen.

**Comment by pld** [comment]
I think yes it does, this was in scope when we talked about the "flavors" earlier. That we'd want custom APKs to only modify the app icon, initial logos and title screen, and be able to set a default app id
